docs/maven-example.md

```diff
@@ -61,19 +68,22 @@ __4. Configure Fabric8 Maven Plugin to use a Docker Compose file__
 
 Add the `<configuration>` and `<executions>` sections to `pom.xml` as shown in above `pom.xml` snippet. Update the `<composeFile>` to provide the relative path of Docker Compose file from `pom.xml`
 
-__5. Deploy application on Kubernetes or OpenShift__
+**5. Deploy application on Kubernetes or OpenShift**
+
+Make sure that Kubernetes/OpenShift cluster or Minikube/minishift is running. In case, if anything of this is not running, you can run minishift to test this application by using following command.
```

Comment:
```suggestion
Make sure that Kubernetes/OpenShift cluster or Minikube/Minishift is running. In case, if anything of this is not running, you can run Minishift to test this application by using the following command.
```